modules/swagger-codegen/src/main/java/com/wordnik/swagger/codegen/languages/ObjcClientCodegen.java

```diff
@@ -66,9 +68,8 @@ public ObjcClientCodegen() {
 
     typeMapping = new HashMap<String, String>();
     typeMapping.put("enum", "NSString");
-    typeMapping.put("Date", "SWGDate");
-    typeMapping.put("DateTime", "SWGDate");
-    // typeMapping.put("Date", "SWGDate");
+    typeMapping.put("Date", "NSDate");
+    typeMapping.put("DateTime", "NSDate");
```

Comment:
we use `NSDate` instead of `SWGDate`, as JSONModel can covert `NSDate` to `NSString` and build
`NSDate` from `NSString` automatically.